### PR TITLE
Changed the checkin script to get the current location from browser.

### PR DESCRIPTION
**Pull request by amshali**

### CODE REVIEW

File: public/javascripts/checkin.js [hunk 1]
```diff
@@ -55,5 +66,6 @@ function initialize() {
                           '<strong>' + place.name + '</strong><br>' + address);
     infowindow.open(map, marker);
   });
+  navigator.geolocation.getCurrentPosition(setCurrentLocation);
```
Comment (ProCynic) [inline public/javascripts/checkin.js:69]: I don't actually see any difference when I run it.

Comment (ProCynic) [inline public/javascripts/checkin.js:69]: never mind, it asked permission and then centered the map over my apartment.

**Comment by ProCynic** [comment]
looks good.

File: public/javascripts/checkin.js (hunk 1 above)
Comment (amshali) [inline public/javascripts/checkin.js:69]: looks good?

Comment (ProCynic) [inline public/javascripts/checkin.js:69]: yep

**Pull request merged**